devel/spicy-analyzers.rst

```diff
@@ -1,35 +0,0 @@
-============================
-Writing analyzers with Spicy
-============================
-
-`Spicy <https://docs.zeek.org/projects/spicy/en/latest/index.html>`_ is a
-parser generator that makes it easy to create robust C++ parsers for network
-protocols, file formats, and more. Zeek supports integrating Spicy analyzers so
-that one can create Zeek protocol, packet and file analyzers.
-
-Spicy is documented separately, so this section just presents high-level points
-relevant for Zeek.
-
-Installation
-============
-
-A Zeek configured with default ``./configure`` options includes Spicy.
-
-Writing an analyzer
```

Comment:
I think this section (especially its first paragraph) was valuable since it gave a quick and high-level introduction. The documentation in this patch instead builds all this up in a very systematic way and only at the very end shows how to actually create a package.

A lot of effort went into the package templates for Spicy analyzers so that it probably is a pretty good starting point. Do you think it would make sense to keep this paragraph as a TLDR; at the top of the Zeek Spicy docs?

Reply:
Makes sense, I've moved that back in.